.github/workflows/python.yml

```diff
@@ -62,7 +62,19 @@ jobs:
     - env:
         SPARK_VERSION: ${{ matrix.spark }}
         SCALA_VERSION: ${{ matrix.scala }}
-      run: if [ ${SPARK_VERSION:0:1} == "3" ]; then mvn -q clean install -DskipTests -Dscala=${SCALA_VERSION:0:4} -Dspark=3.0 -Dgeotools ; else mvn -q clean install -DskipTests -Dscala=${SCALA_VERSION:0:4} -Dspark=2.4 -Dgeotools ; fi
+      run: |
+        if [ ${SPARK_VERSION:0:1} == "3" ]; \
+        then \
+          if [ ${HADOOP_VERSION:0:1} == "3" ]; \
```

Comment:
I don't think `HADOOP_VERSION` is set to anything, these are populated in the `env` block above. And might not be necessary regarding the comment about the new profiles?